build/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
All comments use space between their content and slash characters. Is any reason why you added:
```
//go:build tools
```
instead of
```
// go:build tools
```

Reply:
It is added by `go fmt` starting from Go 1.17.
This is how they should be formated: https://pkg.go.dev/go/build#hdr-Build_Constraints